packages/store/test/test-label-instances.js

```diff
@@ -20,8 +20,7 @@ const DownCounterI = M.interface('DownCounter', {
   decr: M.call().returns(M.number()),
 });
 
-// TODO unskip after reconciling with endo
-test.skip('test defineExoClass', t => {
+test('test defineExoClass', t => {
```

Comment:
BTW next time you might consider using `test.failing()` instead of `test.skip()`, which will not flunk the build until the test [actually starts passing](https://github.com/avajs/ava/blob/main/docs/01-writing-tests.md#failing-tests) , which provides a handy reminder to remove the `.failing`

Reply:
Gotcha. Good to know.